assets/js/modules/analytics-4/components/dashboard/ActivationBanner/ReminderBanner.js

```diff
@@ -158,6 +159,27 @@ export default function ReminderBanner( { onSubmitSuccess } ) {
 		</section>
 	);
 
+	// Show unique errors.
+	const errorNotice =
+		errors &&
+		Object.values( errors )
+			.reduce( ( acc, error ) => {
+				if (
+					! acc.some(
+						( err ) =>
+							err.code === error.code &&
+							err.message === error.message
+					)
+				) {
+					acc.push( error );
+				}
+
+				return acc;
+			}, [] )
+			.map( ( error ) => (
+				<ErrorNotice key={ error.code } error={ error } />
+			) );
```

Comment:
This seems a bit of an oddly specific API. Why not do this entire thing in the parent component (`ActivationBanner`) and instead of having `errors` as a prop that specifically takes error components, use `children` and render whatever is passed to `children` where `errorNotice` in this component is output?

Reply:
Very valid suggestion. Moved it to `ActivationBanner`. Also removed the story now that it is irrelevant.